src/java.base/share/classes/sun/security/ssl/ServerNameExtension.java

```diff
@@ -343,7 +343,7 @@ public void consume(ConnectionContext context,
                         sni, shc.resumingSession.serverNameIndication)) {
                     shc.isResumption = false;
                     shc.resumingSession = null;
-                    // the server is disallowing this session resumption,
+                    // this server is disallowing this session resumption,
                     // so don't include the pre-shared key in the ServerHello handshake message
```

Comment:
If you wouldn't mind <= 80 here also.  The other changes look great.  

Thanks for considering.

Reply:
Sorry Brad, I didn't notice that one in my previous update. I've now updated the PR to fix this line.